website/docusaurus.config.js

```diff
@@ -109,6 +109,16 @@ const config = {
                     src: 'img/logo/logo-no-text.svg',
                 },
                 items: [
+                    {
+                        type: 'docsVersionDropdown',
+                        position: 'right',
+                        dropdownItemsAfter: [
+                            {
+                                href: 'databend-qe5h89pp4-databend.vercel.app',
```

Comment:
Although it is a recent depoly, there are no major changes, just an extra databend-meta-upgrade-09, which I think is reasonable.